core/actionProxy/actionproxy.py

```diff
@@ -251,6 +250,7 @@ def error():
 
 def complete(response):
     # Add sentinel to stdout/stderr
+    print(response)
```

Comment:
Extra logging may cause tests to fail (which might be asserting empty logs ie no message from proxy appearing in action logs).